### PR TITLE
Feat: add testimonials list carousel

### PR DESCRIPTION
**Pull request by abazer27**
## Changes

- feat: add testimonials list carousel

## Preview

- Desktop
image
- Tablet
image
- Mobile
image

## Evidence
title: add testimonials list carousel
project: Desa Digital
participants:  @adzharamrullah @bangunbagustapa

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <template>
   <div>
     <HeroBanner @clickCTA="showModal" />
+    <TestimonialLists />
```
Comment (adzharamrullah) [inline pages/index.vue:4]: this is wrong place component `testimonials`, this is should be above  component `mapsdedi`

File: nuxt.config.js [hunk 2]
```diff
@@ -1,4 +1,7 @@
 export default {
+  server: {
+    host: '0.0.0.0'
```
Comment (adzharamrullah) [inline nuxt.config.js:3]: i think you should remove this, because we not need to configured host

File: windi.config.js [hunk 3]
```diff
@@ -31,5 +33,13 @@ export default {
         800: '#022B55'
       }
     }
-  }
+  },
+  extend: {
+    lineClamp: {
```
Comment (adzharamrullah) [inline windi.config.js:38]: i think this is unnecessary code

File: components/Card/index.vue [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+.navigation{
+  @apply hidden lg:(inline)
+}
+.navigation__wrapper{
+  @apply hidden lg:(inline mt-8)
+}
+.navigation__button-right, .navigation__button-left{
+  @apply w-[42px] h-[42px] rounded-1/2 bg-green-700 text-white
+}
+.swiper-container-horizontal > .swiper-pagination-bullets {
```
Comment (adzharamrullah) [inline components/Card/index.vue:209]: you should give !important every override property style swipper

**Comment by adzharamrullah** [comment]
can you edit the title of Pull Request following the convention name of Pull Request or following the convention name of commit message ?

File: components/TestimonialLists/index.vue [hunk 5]
```diff
@@ -0,0 +1,25 @@
+<template>
+  <div>
+    <Card :testimonials="testimonials" />
+  </div>
+</template>
+
+<script>
+
```
Comment (doohanas) [inline components/TestimonialLists/index.vue:8]: Minor: 

inconsistency, after tag `script` you are not added a new line but in this section you add new line.

File: components/TestimonialLists/index.vue [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    }
+  },
+  async fetch () {
+    const response = await this.$axios.get('/testimonials')
+    this.testimonials = response.data.data
+  }
+}
+</script>
+
+<style>
```
Comment (doohanas) [inline components/TestimonialLists/index.vue:23]: minor: 

 Why did you write this code when it was not used? I think you can remove it for better.

File: components/TestimonialLists/index.vue (hunk 5 above)
Comment (doohanas) [inline components/TestimonialLists/index.vue:8]: Minor: 

inconsistency, you put a new line after tag `script`.

File: components/TestimonialLists/index.vue [hunk 7]
```diff
@@ -0,0 +1,25 @@
+<template>
+  <div>
+    <Card :testimonials="testimonials" />
+  </div>
+</template>
+
+<script>
+
+export default {
+
```
Comment (doohanas) [inline components/TestimonialLists/index.vue:10]: Minor:

inconsistency, you give a line space between `export default` and `data` .

File: components/TestimonialLists/index.vue (hunk 6 above)
Comment (doohanas) [inline components/TestimonialLists/index.vue:23]: minor: 

I think you should remove the unnecessary code when it's not used.

**Review by bangunbagustapa — APPROVED** [review]

**Review by doohanas — APPROVED** [review]

**Review by adzharamrullah — APPROVED** [review]

**Pull request merged**